SoObjects/SOGo/SOGoUserManager.m

```diff
@@ -649,7 +659,7 @@ - (BOOL) checkLogin: (NSString *) _login
             [currentSource setBindPassword: _pwd];
           }
     }
-	    
```

Comment:
https://github.com/Zentyal/sogo/pull/156 I will merge it once the tests pass, and this PR will be updated automatically.